packages/backend/src/core/discovery/DiscoveryRunner.ts

```diff
@@ -40,14 +40,15 @@ export class DiscoveryRunner {
       ? await this.updateInitialAddresses(projectConfig)
       : projectConfig
 
-    const discovery = await this.discover(config, blockNumber)
+    const discovery: DiscoveryOutput = await this.discoverWithRetry(
```

Comment:
I think that this type declaration (`: DiscoveryOutput`) is unnecessary and can be removed